src/index.html

```diff
@@ -12,50 +12,50 @@
         <h1>2048</h1>
         <div class="controls">
           <p class="info">
-            Score: <span class="game-score">0</span>
+            Score: <span id="game-score" class="game-score">0</span>
           </p>
-          <button class="button start">Start</button>
+          <button id="button-start" class="button start">Start</button>
         </div>
       </div>
 
       <table class="game-field">
         <tbody>
           <tr class="field-row">
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
+            <td id="0-0" class="field-cell"></td>
```

Comment:
Adding ids for individual table cells may not be the most efficient way to manage cell data. Consider handling this in JavaScript with an array or object instead.

Reply:
<img width="696" alt="Screenshot 2023-09-04 at 16 31 34" src="https://github.com/mate-academy/js_2048_game/assets/53228259/07ab3514-3a02-41db-8a5f-a9e6671bef7f">

you can use .classList.add(`field-cell--${num}`) and .classList.replace

Reply:
you already do this way, delete all ids